dashboard/config/foorm/forms/surveys/teachers/teacher_end_of_year_survey.json

```diff
@@ -0,0 +1,1060 @@
+     ]
+    },
+    {
+     "type": "html",
+     "name": "input234",
+     "html": "<p><span style=\"background-color: #ffff00;\">TO DO:</span></p>\n<p><span style=\"background-color: #ffff00;\">Add logic based on course to <strong>choose which subcourses or units taught</strong>.</span></p>\n<p><span style=\"background-color: #ffff00;\">CSF: </span></p>\n<p><span style=\"background-color: #ffff00;\">* Course A, B, C, etc.</span></p>\n<p><span style=\"background-color: #ffff00;\">* What type of teacher are you (homeroom, library, tech, etc)</span></p>\n<p><span style=\"background-color: #ffff00;\">CSD: </span></p>\n<p><span style=\"background-color: #ffff00;\">* Unit 1, Unit 2, Unit 3, ...etc.</span></p>\n<p><span style=\"background-color: #ffff00;\">CSP: </span></p>\n<p><span style=\"background-color: #ffff00;\">* Unit 1, Unit 2, UNit 3, ...etc.</span></p>\n<p><span style=\"background-color: #ffff00;\">* as AP, honors, regular.</span></p>"
+    },
+    {
+     "type": "checkbox",
+     "name": "csf_which_course",
+     "title": "***IF CS Fundamentals*** Which courses of CS Fundamentals did you teach? Please check any courses you taught all or most of with students.",
+     "choices": [
+      "Course A",
+      "Course B",
+      "Course C",
+      "Course D",
+      "Course E",
+      "Course F",
+      "Course 1",
+      "Course 2",
+      "Course 3",
+      "Course 4",
+      "Express",
+      "20-hour"
+     ]
+    },
+    {
+     "type": "checkbox",
+     "name": "csd_which_units",
+     "title": "***IF CS DISCOVERIES*** Which units of CS Discoveries did you teach? Please check any units you taught all or most of.",
+     "choices": [
+      "Unit 1 - Problem Solving",
+      "Unit 2",
+      "Unit 3",
+      "Unit 4",
+      "Unit 5",
+      "Unit 6"
+     ]
+    },
+    {
+     "type": "checkbox",
+     "name": "csp_which_units",
+     "title": "***IF CS PRINCIPLES*** Which units of CS Pprinciples did you teach? Please check any units you taught all or most of.",
+     "choices": [
+      "Unit 1 - The Internet",
+      "Unit 2",
+      "Unit 3",
+      "Unit 4",
+      "Unit 5",
+      "Explore-PT Prep",
+      "Create PT Prep"
+     ]
+    },
+    {
+     "type": "checkbox",
+     "name": "csp_taught_as_ap",
+     "title": "***IF CS PRINCIPLES*** This year I taught CS Principles as...[check all that apply]",
+     "choices": [
+      "An AP course",
+      "An honors course (non-AP)",
+      "A regular course (non-AP)"
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "course_required",
+     "title": "At your school is {course} a required or elective course?  ",
+     "choices": [
+      "Required (or part of a rotation that all students do at some point)",
+      "Elective",
+      "Integrated into other subjects or school activities."
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "years_taught_codeorg",
+     "title": "How many years have you taught using the Code.org {csp_or_csd} curriculum?",
```

Comment:
will you want a variable called csp_or_csd or is this a placeholder?

Reply:
It's an artifact.  should be replaced with {course}.   will do.